jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -244,21 +264,42 @@ def render(
         title="Created Job Script",
     )
 
-    # `submit` will be `None` --submit/--no-submit flag was not set
-    if submit is None:
-        # If not running in "fast" mode, ask the user what to do.
-        if not fast:
-            submit = typer.confirm("Would you like to submit this job immediately?")
+    # `submit` will be `None` when --submit/--no-submit flag was not set
+    if submit is None and (cluster_name or execution_directory):
+        # If the user specified --cluster-name or --execution-directory, they must also specify --submit
+        raise Abort(
+            "You must specify --cluster-name and --execution-directory only on submit mode.",
+            subject="Incorrect parameters",
+            support=True,
+        )
 
-        # Otherwise, assume that the job script should be submitted immediately
-        else:
-            submit = True
+    """
+    Invoke `question_helper` to handle the logic for parameters that are not specified by the user.
+
+    If the user specified --fast, the `question_helper` will not invoke the function and will
+    instead return the default answer. If the user did not specify --fast, the `question_helper`
+    will invoke the function and return the user's answer.
+    """
+
+    submit = question_helper(typer.confirm, "Would you like to submit this job immediately?", True, fast, submit)
```

Comment:
The helper was created because we had to ask multiple questions to the user, which was making the logic in the method a lot more confusing.

Following @fschuch's suggestion, I'll add questions for `cluster_name` and `execution_directory`, even though they have default values at the submittion level, to give the user a chance to specify it in non fast mode.

This will make the helper be used 4x. Do you think it's still worth putting the logic in the method?

Reply:
I'm ok with it. Good touch modifying the notation to keyword arguments, I think that clarifies the topic